examples/r2i/meson.build

```diff
@@ -0,0 +1,10 @@
+# Define common headers directory for examples
+common_inc_dir = include_directories('common')
```

Comment:
Why do we need this?

Reply:
What is in common that we need to include?

Reply:
As mentioned [here](https://github.com/RidgeRun/r2inference/pull/77#discussion_r420455610) examples need the stb headers in the examples common directory (just in case, this is not the common directory of the project itsefl)